### PR TITLE
Build classifier

### LINKED ISSUE #31: ML Model Implementation
The next step to take after #10 . There is an idea of the dimensionality of features/labels. So that shall be enough to create a model prototype, based on the research. The model can be created by making use of [pipelines](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html) and [subclassing base classes](https://scikit-learn.org/stable/developers/develop.html#rolling-your-own-estimator), to package the model into an sklearn-like class.

### PR DESCRIPTION
**Pull request by VasLem**
Closes #31 and #64

### CODE REVIEW

**Comment by VasLem** [comment]
@bajo1207 @antosalerno @OlympiaG After having experimented with different models, we came to the understanding that xgboost regressor performs best. We can merge it

**Review by antosalerno — APPROVED** [review]
